src/pages/components/accordion/code.mdx

```diff
@@ -5,6 +5,16 @@ tabs: ['Code', 'Usage', 'Style', 'Accessibility']
 ---
 
 <ComponentDemo
+  items={[
```

Comment:
We might wanna come up with a more descriptive prop that icons `components` maybe?